pkg/chart/models/chart.go

```diff
@@ -43,7 +43,7 @@ type RepoInternal struct {
 
 // Chart is a higher-level representation of a chart package
 type Chart struct {
-	ID              string             `json:"ID" bson:"_id"`
+	ID              string             `json:"ID" bson:"ID"`
```

Comment:
`bson:"id"`? (even though I think this is not case sensitive) but as I said before, using just `id` may be tricky, I would change it more